### PR TITLE
Unify Collect.js examples

### PR DESCRIPTION
**Pull request by opeida**
Currently, Collect.js has ~19 used examples. The plan is to move all of them to GitHub and link to a single online code editor.
[Slack thread](https://getverygood.slack.com/archives/C01L35KS3L2/p1618340837001400)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,31 +20,31 @@ Here we've collected examples of library use cases:
 
 ### 1. Capturing credit card data
 
-[Code example is here](examples/v2/credit-card-example)
+[Code example is here](examples/1-usecases/credit-card-example)
```
Comment (ilovetea) [inline README.md:23]: what is the reason for `1-` prefix? can we simplify?

Comment (opeida) [reply README.md:23]: The reason is to sort examples from the most general to the most specific, similar to documentation flow.
image

Comment (ilovetea) [reply README.md:23]: I would remove numbers they're a bit confusing also what if decide to change general / specific things or add smth in between. Can we remove numbers?

**Comment by sanohin** [comment]
1. I noticed we have different code style in examples, can you please run `prettier` to format everything. [Docs here]( https://prettier.io/docs/en/install.html). I think you should run `npx prettier --write .` in the project root
2. I think we can't solve this easily but maybe there is a feature in GitHub to include a markdown file in another one? We have plenty of the same files. It'd be cool to embed one README in all the example README files.

File: README.md (hunk 1 above)
Comment (flor-master) [reply README.md:23]: Agree - we need to remove digits

**Comment by AnnaKudriasheva** [comment]
Before PR merge we need to check cross-links from the Dashboard/Documentation since we've changes some URLs for our examples.

**Review by AnnaKudriasheva — APPROVED** [review]

**Review by ilovetea — APPROVED** [review]

**Comment by opeida** [comment]
> Before PR merge we need to check cross-links from the Dashboard/Documentation since we've changes some URLs to our examples.
[Update links to Collect.js examples #1302](https://github.com/verygoodsecurity/docs/pull/1302)
The only links that have to be changed along with the current PR

**Pull request merged**